src/DevTools/UserAccess.php

```diff
@@ -50,6 +63,14 @@ public function register() {
 	 * @return bool Whether developer tools are enabled for the user.
 	 */
 	public function is_user_enabled( $user = null ) {
+		// Note: This is somewhat of a shortcut for blocking access to the validation UI on unsupported versions of WP.
+		// It works because we're already using this method in many places to check if the user interface should be
+		// shown, and its the user interface which is particularly problematic on older versions of WP due to the
+		// JavaScript dependencies.
+		if ( ! $this->dependency_support->has_support() ) {
+			return false; // @codeCoverageIgnore
```

Comment:
> I think we should just prevent users access to validation screens in WP<5.6.

@westonruter Actually, in light of https://github.com/ampproject/amp-wp/issues/6775#issuecomment-989305532, do you think we should disable developer tools on WP<5.6, too? If yes, then we should hide the toggle here and also remove one of the steps in the Onboarding Wizard.

Reply:
That's a good question. Developer tools access is broken down into two aspects:

* Capability check: the users should never have access to DevTools (or the validation nonce)
* Preference check: validation data should not be presented to the user.

In the case of a user being on an old WP install, the latter check is more appropriate because what we're wanting to do is to prevent displaying the validation _UI_ to the user since it won't work properly. Technically such a user could still manually go to the validation screens if they know the URLs, but that is OK since they still have the capability and we aren't blocking their access.

So I think we can keep the step in the Onboarding Wizard. It's just we won't link to the Validated URLs screen.

Reply:
Oh! I see what you're referring to.

Reply:
I've addressed this in ff7e7b783 by hiding the toggle which appears on the settings screen as well as the toggle that appears on the user profile screen.

Reply:
> also remove one of the steps in the Onboarding Wizard.

@delawski Oh, I didn't do this part. If you would open a PR to address this, I'd appreciate it. It's consider this not to be a blocker for 2.2, since it only affects WP<5.6 (which granted is about a third of all installs).

Reply:
Sure! This one seems to be easy: #6789.